rust_src/src/marker.rs

```diff
@@ -30,39 +30,39 @@ impl LispMarkerRef {
         unsafe { mem::transmute(self.as_ptr()) }
     }
 
```

Comment:
I am not opposed to moving to `isize`. However, we should be consistent. Does this mean we also drop the `c_int` usage as well?

Reply:
I thought we only use C types when rust functions are used by C functions.